sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed bug where calling StopAsync in QueueListener while drain mode was enabled would cancel the execution cancellation token.
+- In QueueListener When ProcessMessageAsync calls QueueProcessor.CompleteProcessingMessageAsync, pass in only the shutdown cancellation token only and not the caller's cancellation token in order to allow for graceful shutdown.
```

Comment:
Trying to align my understanding here. But I think this should be changed to

"Fixed bug where the cancellation token passed to the QueueListener.ProcessMessageAsync, was being passed to the QueueProcessor.CompleteProcessingMessageAsync call, which if the token was cancelled would not allow the message to be deleted".

Reply:
Yes I think that's a clear way to explain it thanks :)